src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTestForFailingNotification.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.tasks;
+
+import com.microsoft.azure.storage.blob.CloudBlockBlob;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.bulkscanprocessor.config.IntegrationTest;
+import uk.gov.hmcts.reform.bulkscanprocessor.model.out.msg.ErrorCode;
+import uk.gov.hmcts.reform.bulkscanprocessor.model.out.msg.ErrorMsg;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.verify;
+import static uk.gov.hmcts.reform.bulkscanprocessor.helper.DirectoryZipper.zipDir;
+import static uk.gov.hmcts.reform.bulkscanprocessor.model.common.Event.FILE_VALIDATION_FAILURE;
+import static uk.gov.hmcts.reform.bulkscanprocessor.model.common.Event.ZIPFILE_PROCESSING_STARTED;
+
+@IntegrationTest
+@RunWith(SpringRunner.class)
+@TestPropertySource(properties = {
+    "containers.mappings[0].container=bulkscan",
+    "containers.mappings[0].jurisdiction=BULKSCAN",
+    "containers.mappings[0].poBox=BULKSCANPO",
+    "containers.mappings[0].paymentsEnabled=false",
+    "containers.mappings[0].enabled=false"
+})
+public class BlobProcessorTaskTestForFailingNotification extends ProcessorTestSuite<BlobProcessorTask> {
```

Comment:
By convention, `Test` should go to the end of the test class name.

Reply:
There are quite a lot of test classes in this package named BlobProcessorTaskTestFor..., and this approach makes sense in my opinion

Reply:
we have few of those. i guess shouldn't be harmful here either

Reply:
Aaa, I see, and all of them are for BlobProcessorTask. Fair enough. Might need to rename them all on one go later.